tests/common.py

```diff
@@ -160,6 +160,7 @@ def no_windows(note=''):
 def requires_native_clang(func):
   assert callable(func)
 
+  @wraps(func)
```

Comment:
What is this?

Reply:
I don't know, but we use it elsewhere and without it the order of decorators on the tests seems to matter, which is not what we want. In particular, adding this let me use the `@parameterized` decorator last instead of first.